Gopkg.toml

```diff
@@ -64,7 +64,7 @@ required = [
   unused-packages = true
 
 [[constraint]]
-  branch = "master"
+  branch = "oidc-provisioner"
```

Comment:
let's just remember to change this. I always feel like I'll forget.

Reply:
👍